sherpa/utils/logging.py

```diff
@@ -37,3 +37,26 @@ def config_logger(name, level=logging.WARNING, stream=sys.stdout, template=None)
     logger.propagate = False
 
     return logger
+
+
+class SherpaVerbosity():
```

Comment:
Should this derive from contextlib.AbstractContextManager (now we're safe to require python 3.6)? To be honest it's not obvious from the documentation what benefits we get, but I do like the implied contract/documentation you get from something like this.

Reply:
Sure. We can do that. Maybe some benefit will be added in later Python versions...